cscs-checks/tools/profiling_and_debugging/src/roofline/LICENSE

```diff
@@ -0,0 +1,254 @@
+?commercial computer software documentation? as specified under FAR 12.212 (48
+C.F.R. 12.212) or DFARS 227.7202 (48 C.F.R. 227.7202), as applicable. This
+commercial computer software and related documentation is provided to end users
+for use by and on behalf of the U.S. Government, with only those rights as are
+granted to all other end users pursuant to the terms and conditions herein. Use
+for or on behalf of the U.S. Government is permitted only if the party
+acquiring or using this software is properly authorized by an appropriate U.S.
+Government official. This use by or for the U.S. Government clause is in lieu
+of, and supersedes, any other FAR, DFARS, or other provision that addresses
+Government rights in the computer software or documentation covered by this
+license. All copyright licenses granted to the U.S. Government are coextensive
+with the technical data and computer software licenses granted herein. The U.S.
+Government will only have the right to reproduce, distribute, perform, display,
+and prepare derivative works as needed to implement those rights.
+
+10. APPLICABLE LAWS: All disputes arising out of or related to this Agreement,
+whether based on contract, tort, or any other legal or equitable theory, will
+in all respects be governed by, and construed and interpreted under, the laws
+of the United States of America and the State of Delaware, without reference to
+conflict of laws principles. The parties agree that the United Nations
+Convention on Contracts for the International Sale of Goods (1980) is
+specifically excluded from and will not apply to this Agreement. All disputes
+arising out of or related to this Agreement, whether based on contract, tort,
+or any other legal or equitable theory, will be subject to the exclusive
+jurisdiction of the courts of the State of Delaware or of the Federal courts
+sitting in that State. Each party submits to the personal jurisdiction of those
+courts and waives all objections to that jurisdiction and venue for those
+disputes.
+
+11. SEVERABILITY: The parties intend that if a court holds that any provision
+or part of this Agreement is invalid or unenforceable under applicable law, the
+court will modify the provision to the minimum extent necessary to make it
+valid and enforceable, or if it cannot be made valid and enforceable, the
+parties intend that the court will sever and delete the provision or part from
+this Agreement. Any change to or deletion of a provision or part of this
+Agreement under this Section will not affect the validity or enforceability of
+the remainder of this Agreement, which will continue in full force and effect.
+
+12. EXPORT: You must comply with all laws and regulations of the United States
+and other countries governing the export, re-export, import, transfer,
+distribution, use, and servicing of Software. In particular, You must not: (a)
+sell or transfer Software to a country subject to sanctions, or to any entity
```

Comment:
I think GitHub does not block access to this file based on the countries that are subject to the United Stated government list of denial. So, I think we cannot publish this test.

Reply:
can you review https://github.com/eth-cscs/reframe-tests/pull/51 ?

Reply:
Is this expected ?

Reply:
> wget https://github.com/eth-cscs/reframe-tests/archive/v2.16.tar.gz

```
--2019-03-28 11:38:01--  https://github.com/eth-cscs/reframe-tests/archive/v2.16.tar.gz
Resolving github.com (github.com)... 140.82.118.3, 140.82.118.4
Connecting to github.com (github.com)|140.82.118.3|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2019-03-28 11:38:01 ERROR 404: Not Found.
```